src/lib.rs

```diff
@@ -1,13 +1,40 @@
+//! Virtual touch screen analog joysticks for the Bevy game engine.
+//!
+//! see the examples for more detailed usage
+//!
+//! ## Example
+//!```no_run
```

Comment:
```suggestion
//!```no_run
// # use bevy::prelude::*;
// use bevy_touch_stick::prelude::*
```

If you start lines with hash, they will not appear in the documentation.

Maybe I missed something here, just wanted to show you the idea.

Reply:
the idea i was going for in the example was just a simple thing you can copy paste into the code
i dont really know if it makes sense too do so, but i was trying too copy bevys readme example but i did not want too put 200 lines of garbage at the top of lib.rs :laughing: